test/spec/tagspec.js

```diff
@@ -86,6 +86,14 @@ describe("Cloudinary.ImageTag", function() {
     tag = new cloudinary.ImageTag('image_id', config).toHtml();
     return expect(tag).toBe(`<img src="${DEFAULT_UPLOAD_PATH}image_id">`);
   });
+  it("should create an image tag with encoded special chars", function() {
+    let imageId = `test's special < \"characters\" >`;
+    let encodedImageId = encodeURIComponent(imageId);
```

Comment:
@maoznir, using the same function that the tag creation is using to create the expected value is problematic. 
If there's a bug in the function, this test will still pass.  
Please put the actual value you are expecting to get

Reply:
done